### DISCUSSION TITLE
Multiple brave profiles?

### DISCUSSION THREAD

**Discussion by berters** [Acidham/chromium-hist-bookmarks#18] [Q&A] [answered]
Is there a way to make this work with multiple Brave profiles? I think currently it just pulls from one. I use two - one for personal, one for work. Thanks!

**Comment by Acidham** [comment]
Sure it is possible, but I do not use profiles and I have no idea on how profiles work. But happy to accept PRs

**Reply by berters** [reply]
I'm afraid I'm a user without technical ability.

Do you think it's feasible to be able to set the default profile somehow? I use two profiles and currently the workflow is using the one I don't want.

Thanks!

**Comment by Acidham** [comment, marked as answer]
The workflow is picking up the `Default` profile because new profiles will be created with `Profile 1` etc. 

If you would like to hack it, open `chrom_history.py` in workflow directory and change URL under `brave` by just changing `Default` to `Profile 1` (or whatever your profile folder is named)

**Reply by berters** [reply]
Thank you! This is perfect :)
